doc/source/notebooks/basics/GPLVM.ipynb

```diff
@@ -62,7 +62,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Following the GPflow notation we assume this dataset has size [num_data, output_dim]"
+    "Following the GPflow notation we assume this dataset has a size of `[num_data, output_dim]`"
```

Comment:
```suggestion
    "Following the GPflow notation we assume this dataset has a shape of `[num_data, output_dim]`"
```
(size is generally the total number of elements, e.g. num_data * output_dim in this case...)